### PR TITLE
BaSD-MS-36: Block buttons on loading

### PR DESCRIPTION
**Pull request by ignacioaristo**
A component `IsLoading ` was created, it shows when the servers need to retrieve data and disappear when the call ends.

image

### CODE REVIEW

**Review by leimeterjoaquin — APPROVED** [review]

File: src/Components/Admins/Form/index.js [hunk 1]
```diff
@@ -100,6 +104,10 @@ const Form = () => {
     setShowErrorMessage(false);
   };
 
+  if (isLoading) {
+    return <IsLoading />;
+  }
+
```
Comment (aquilesmariani) [inline src/Components/Admins/Form/index.js:110]: I don't think curly braces are needed for this case
```suggestion
  if (isLoading) return <IsLoading />;
```

File: src/Components/Admins/index.js [hunk 2]
```diff
@@ -62,6 +68,10 @@ const Admins = () => {
     setShowErrorMessage(false);
   };
 
+  if (isLoading) {
+    return <IsLoading />;
+  }
+
```
Comment (aquilesmariani) [inline src/Components/Admins/index.js:74]: Same as above

File: src/Components/Applications/index.js [hunk 3]
```diff
@@ -37,7 +42,8 @@ function Applications() {
         closeModal();
         setApplications(applications.filter((a) => a._id !== idApp));
       })
-      .catch((error) => error);
+      .catch((error) => error)
```
Comment (aquilesmariani) [inline src/Components/Applications/index.js:45]: Can you show the error in the modal? Seems like this got merged into development without a fix

**Review by aquilesmariani — CHANGES_REQUESTED** [review]
Some tweaks to go.

File: src/Components/Applications/index.js (hunk 3 above)
Comment (leimeterjoaquin) [reply src/Components/Applications/index.js:45]: All this issues are fixed in MS-34: Modal

**Review by aquilesmariani — APPROVED** [review]
👌

**Pull request merged**